### ISSUE TITLE
StatsCAN (3.0.2): Remove blue box (iOS)/circle (android) and * where number on notification 

### ISSUE THREAD

**Issue by levegab** [StatCan/StatcanPublicationUAT#497]
Where there is currently a blue box (ios)/circle (android) and * showing where number on notification bell, it should be a number. Currently experiencing this on the Publications screen [All and For You tabs]).

**Comment by levegab** [comment]
![image](https://user-images.githubusercontent.com/106617357/197194563-d182664c-4ca3-439c-b34f-5f53c1ce580a.png)

**Comment by levegab** [comment]
![Screenshot_20221021-092012_StatsCAN~2](https://user-images.githubusercontent.com/106617357/197195027-8830c109-11c4-4d01-96fd-797e06908362.jpg)

On Android

**Comment by weglmar** [comment]
I'm unsure if this is a UX ticket or a bug that was caused from another feature being implemented. @zhuelle can you please confirm why the * is appearing, and on iOS devices this is appearing as a box? 
Enhancements to the notification bell and screen are coming in #473.

**Comment by zhuelle** [comment]
@weglmar This ticket created by @levegab. We can clarify this ticket at our project meeting.

**Comment by zhuelle** [comment]
@Rajat-Sen Please see #2 item in Jiandong's comment above.  This should be resolved in Enhancements to the notification bell and screen are coming in https://github.com/StatCan/StatcanPublicationUAT/issues/473. Thanks!

**Comment by Rajat-Sen** [comment]
As per our Notifications requirements, we won't be exceeding more than 25 notifications and the it should still look fine. If needed we can reduce the size of text which shows the number of notifications. 

Note to consider: We can add + sign if there are a lot of notifications. Let me know your thoughts!

**Comment by zhuelle** [comment]
I like the idea of adding + sign if there are a lot of notification such as 25+. 25 is the biggest number that we display for total of notification. This should be ironed out during specification on #473. 
CC: @weglmar and @radwahm

**Comment by radwahm** [comment]
This issue is ironed out in https://confluenceb.statcan.ca/display/SPMA/Feature+3.1+Changes+to+the+Notifications+screen 
Github ticket https://github.com/StatCan/StatcanPublicationUAT/issues/473. 

Please ensure we do not begin development on this ticket before 473 is signed off.

**Comment by Jiandong-Guo** [comment]
Please test in next releasae

**Comment by levegab** [comment]
Ticket closed by mistake.

**Comment by weglmar** [comment]
@ForumG-SC When clicking the notification bell to view the notifications, the number disappears (intended). However, if I force close the app and reopen, the number of notifications reappears. Is there a way to have the number not reappear again? (unless new notifications have been added). 

Also, the colour of the bell will need to change in dark mode and light mode (should this be a new ticket?) We will also need to get the colour from Rajat.

**Comment by zhuelle** [comment]
@weglmar Would you please review Forum's comments above and let us know if you have any questions and if it should be included in the Production release? Thanks!

**Comment by weglmar** [comment]
@zhuelle @ForumG-SC this can remain as is for this release! 

The colour of the bell in dark mode would still need to change. I can create a new ticket for this if needed? If not, the colour code should be: #0078D7

**Comment by zhuelle** [comment]
@weglmar I would recommend to create a new ticket for the color change and keep the existing ticket open as the issue is not resolved. Thanks!
